scripts/update_config.js

```diff
@@ -22,7 +22,7 @@
 const fs = require('fs');
 const path = require('path');
 const program = require('commander');
-const merge = require('lodash/merge');
+const _ = require('lodash');
```

Comment:
you can still do `const { merge, set, get } = require('lodash');` as you were intending to do `const merge = require('lodash/merge');` before.

Reply:
But I don't want to have `set`, `get` as seperate function. Looks less readable than `_.set(`

Reply:
:) so what was the intention behind `const merge = require('lodash/merge');`?

Reply:
Yes that time we need only `merge` from lodash. But now as we need more functions from it, we can use the library directly.